pkg/bpf/map_linux.go

```diff
@@ -908,6 +969,8 @@ func (m *Map) DeleteAll() error {
 			log.WithError(err2).Warningf("Unable to correlate iteration key %v with cache entry. Inconsistent cache.", nextKey)
 		}
 
+		defer m.History.addDeleteEntry(mk, err)
```

Comment:
It would be better to perform some kind of aggregation. If the map has 10000 entries, we will be calling `addDeleteEntry` 10000 times although we only care the last 50. The aggregation could be done on a per `DeleteAll()` function call where we would store the number of entries deleted and store it in the `m.History`.

Reply:
I have (probably) even a better idea - when we delete all entries in the map, maybe let's just leave it as one information with key value `ALL`? Which would mean that we performed deletion of all keys.

I pushed that change. Let me know if that approach is OK for you.